crates/proof-of-sql/tests/timestamp_integration_tests.rs

```diff
@@ -342,3 +422,99 @@ mod tests {
         );
     }
 }
+
+#[test]
+#[cfg(feature = "blitzar")]
+fn we_can_prove_timestamp_inequality_queries_with_multiple_columns() {
+    use curve25519_dalek::RistrettoPoint;
+    use proof_of_sql::base::{database::OwnedTable, scalar::Curve25519Scalar};
+
+    let mut accessor = OwnedTableTestAccessor::<InnerProductProof>::new_empty_with_setup(());
+    accessor.add_table(
+        "sxt.table".parse().unwrap(),
+        owned_table([
+            timestamptz(
+                "start_time",
+                PoSQLTimeUnit::Nanosecond,
+                PoSQLTimeZone::Utc,
+                [
+                    0, // Beginning of Unix time
+                    444_972_800_000_000_000,
```

Comment:
I'd make another row non-selected so you may as well change just one row a little bit